_tags/iteration.md

```diff
@@ -127,21 +148,43 @@ Begins the loop at the specified index.
 3 4 5 6
 ```
 
+To start a loop from where the last loop using the same iterator left off, pass the special word `continue`.
+
+<p class="code-label">Input</p>
+```liquid
+{% raw %}
+<!-- if array = [1,2,3,4,5,6] -->
```

Comment:
I've skipped converting all this page's examples to dynamic to keep them understandable at a glance. Otherwise they add extra line breaks to an already-long page, or obscure their meaning by attempting to manage the whitespace.

Reply:
I'm OK with leaving it, then. 👍